src/test/java/com/google/api/codegen/gapic/testdata/nodejs/nodejs_library.baseline

```diff
@@ -778,7 +778,7 @@ function samplePublishSeries(shelfName, edition) {
       const bookNames = response.bookNames;
       for (const title of response.bookNames) {
         // Now we deal with thisSingleBook!
-        console.log(`Title: {$title}`);
+        console.log(`\t% The book's title: "{$title}", \\\nand the book costs $50.00 %`);
```

Comment:
We use backticks? Then we need to escape any backticks, too.

And a few lines below, we have single-quoted default values. We should makes sure to escape single quotes in that case.